### ISSUE TITLE
Php notices at clean install, due to absence of a podmotor_account_id

### ISSUE THREAD

**Issue by maartenhunink** [TheCraigHewitt/Seriously-Simple-Podcasting#443]
After installing the plugin and trying to add an episode, you get the following notices on the Add new post page.
```
Notice: Trying to access array offset on value of type bool in /test/wp-content/plugins/seriously-simple-podcasting/php/includes/ssp-functions.php on line 870

Notice: Trying to access array offset on value of type bool in /test/wp-content/plugins/seriously-simple-podcasting/php/includes/ssp-functions.php on line 871

Notice: Trying to access array offset on value of type bool in /test/wp-content/plugins/seriously-simple-podcasting/php/includes/ssp-functions.php on line 872

Notice: Trying to access array offset on value of type bool in /test/wp-content/plugins/seriously-simple-podcasting/php/includes/ssp-functions.php on line 873
```

Make sure `define( 'WP_DEBUG', true );` is set in wp-config.php when testing.

**Comment by jonathanbossenger** [comment]
Thanks @maartenhunink we have a plugin development sprint coming up soon, I'll include this and #444 in that sprint.

**Comment by maartenhunink** [comment]
Awesome. Thank you!!

**Comment by jonathanbossenger** [comment]
This has been fixed in the 2.0.0 release
